deploy/docker/docker-entrypoint.sh

```diff
@@ -177,10 +178,11 @@ if [[ ! -z "$EMQX_LOADED_MODULES" ]]; then
     echo "EMQX_LOADED_MODULES=$EMQX_LOADED_MODULES"
     # Parse module names and place `{module_name, true}.` tuples in `loaded_modules`.
     for var in $(echo "$EMQX_LOADED_MODULES"|sed -e "s/^[^A-Za-z0-9_]\{1,\}//g"|sed -e "s/[^A-Za-z0-9_]\{1,\}/\ /g"); do
-        if [ ! -z $(grep -o $var ${_EMQX_HOME}/data/loaded_modules) ]; then
+        if [ ! -z "$(grep -oE "\{($var),[ ]*(true|false)\}" ${_EMQX_HOME}/data/loaded_plugins)" ]; then
```

Comment:
isn't this supposed to be `loaded_modules` not `loaded_plugins`?

Reply:
Hi, @louisburton
Thanks for your feedback, I will fix it in https://github.com/emqx/emqx-rel/pull/561 and https://github.com/emqx/emqx-rel/pull/559